### PR TITLE
Make the web site a static website + create guide's pdf version

### PR DESCRIPTION
**Pull request by anediaz**

### CODE REVIEW

**Comment by goldbergyoni** [comment]
@anediaz @rluvaton WOW, this looks impressive and comprehensive, shall I review or @rluvaton who is more down to the details?

**Comment by anediaz** [comment]
> @anediaz @rluvaton WOW, this looks impressive and comprehensive, shall I review or @rluvaton who is more down to the details?

Yes please!
Maybe @rluvaton is the best to review it, but I am sure it'll be OK if you review it :-)

**Comment by rluvaton** [comment]
I will review it very soon could you just add your docusarus config file?

**Comment by anediaz** [comment]
> I will review it very soon could you just add your docusarus config file, package.json and other necessary files? So I can build it on my machine?

Oh yes, of course, I forgot it!

**Comment by rluvaton** [comment]
I used my config and sadly it failed to build, some of the reasons are:
1. Reference the assets like it in the current directory
2. Forgot to change the reference to the translation files (`README.fr.md` instead of `../../../fr/README.md`)
3. The English README.md file was in the section folder


Other notes:
In bullet point 1.1 in (I think) every language readme,

File: i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md [hunk 1]
```diff
@@ -10,16 +10,16 @@ O APM (monitoramento de desempenho de aplicação) refere-se a uma família de p
 
 ### Exemplo de APM - um produto comercial que visualiza o desempenho de aplicações de serviço cruzado
 
-![Exemplo de APM](/assets/images/apm1.png "Exemplo de APM")
+![Exemplo de APM](./assets/images/apm1.png)
```
Comment (rluvaton) [inline i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:13]: accidentally removed

File: i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md [hunk 2]
```diff
@@ -10,16 +10,16 @@ O APM (monitoramento de desempenho de aplicação) refere-se a uma família de p
...
 
-![Exemplo de APM](/assets/images/apm1.png "Exemplo de APM")
+![Exemplo de APM](./assets/images/apm1.png)
 
 <br/><br/>
 
 ### Exemplo de APM - um produto comercial que enfatiza a pontuação da experiência do usuário
 
-![Exemplo de APM](/assets/images/apm2.png "Exemplo de APM")
+![Exemplo de APM](./assets/images/apm2.png)
```
Comment (rluvaton) [inline i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:19]: accidentally removed

File: i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md (hunk 1 above)
Comment (anediaz) [reply i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:13]: Has this happened due to the absolute path converting script?
Should I rollback everything?

Comment (rluvaton) [reply i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:13]: I think I'll build a migration script that gonna use markdown parsing which is more reliable than my replace script

Comment (rluvaton) [reply i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:13]: @anediaz I've created the repo for the migration script and send you an invite in the email to be a collaborator.

The repo is: [rluvaton/migrate-nodebestpractices-to-docusaurus](https://github.com/rluvaton/migrate-nodebestpractices-to-docusaurus)

I've explained everything there, here's the [link](https://github.com/rluvaton/migrate-nodebestpractices-to-docusaurus/discussions/1#discussion-3425263)

Comment (anediaz) [reply i18n/br/docusaurus-plugin-content-docs/current/sections/production/apmproducts.md:13]: I've just accepted your invitation!
I could take a look to your repo tomorrow, this is an excellent initiative!

Thank you once again Raz!

**Comment by goldbergyoni** [comment]
@anediaz @rluvaton If you anything from me, I'm here, looks like your process is smooth

**Comment by goldbergyoni** [comment]
@anediaz @rluvaton I saw the progress today and was thrilled! Beautiful web site, tres jolie 😍

I bought a netlify pro account for us and also plan to buy a domain. Which one? nodejsbestpractices.com, .io?

**Comment by rluvaton** [comment]
> @anediaz @rluvaton I saw the progress today and was thrilled! Beautiful web site, tres jolie 😍
> 

Thank you!


> I bought a netlify pro account for us and also plan to buy a domain. Which one? nodejsbestpractices.com, .io?

You can `.com`, but both options are great!

**Comment by josh-hemphill** [comment]
> I bought a netlify pro account for us and also plan to buy a domain. Which one? nodejsbestpractices.com, .io?

I went around looking at domains a while ago and found that nodejs.best is available

**Comment by goldbergyoni** [comment]
> > I bought a netlify pro account for us and also plan to buy a domain. Which one? nodejsbestpractices.com, .io?
> 
> I went around looking at domains a while ago and found that nodejs.best is available

Just bought nodejsbestpractices.com and then saw this message... nodejs.best sounds cool

**Comment by rluvaton** [comment]
I think we can close this as I separated the migrate into multiple little PRs (that most of them already merged)

**Pull request closed without merging**